File: Calculator.js
var args = process.argv[2].split(" ")
var regExpr = new RegExp('\\d*(\\.\\d|\\d\\.)\\d*');
var operation = args[1]
var output
if(regExpr.test(args[0]) || regExpr.test(args[2])){
	var int1 = parseFloat(args[0])
	var int2 = parseFloat(args[2])
}
else{
	var int1 = parseInt(args[0])
	var int2 = parseInt(args[2])
}
if(operation === '+'){
	output = int1 + int2
}
else if(operation === '-'){
	output = int1 - int2
}
else if(operation === '*'){
	output = int1 * int2
}
else{
	output = int1 / int2
}
output.toFixed(2)
var result = process.argv[2] + ' = ' + output
console.log(result)

File: Calculator.py
import sys
import re

def main(argv):
	text = argv[1]
	args = text.split(" ")
	regExpr = "\\d*(\\.\\d|\\d\\.)\\d*"
	x = re.search(regExpr, args[0])
	y = re.search(regExpr, args[2])
	if x or y:
		int1 = float(args[0])
		int2 = float(args[2])
		if args[1] == "+":
			output = int1 + int2
		elif args[1] == "-":
			output = int1 - int2
		elif args[1] == "*":
			output = int1 * int2
		else:
			output = int1 / int2
		print(argv[1], "=", output)
	else:
		int1 = int(args[0])
		int2 = int(args[2])
		if args[1] == "+":
			output = int1 + int2
		elif args[1] == "-":
			output = int1 - int2
		elif args[1] == "*":
			output = int1 * int2
		else:
			output = int1 / int2
		print(argv[1], "=", "{:.0f}".format(output))
	
main(sys.argv)